fastlane/Fastfile

```diff
@@ -58,7 +58,8 @@ platform :ios do
         workspace: ENV["WORKSPACE_PATH"],
         configuration: ENV["CONFIGURATION"],
         scheme: ENV["SCHEME"],
-        output_name: ENV["OUTPUT_PATH"],
+        output_directory = ENV["OUTPUT_DIRECTORY"],
+        output_name: ENV["OUTPUT_NAME"],
```

Comment:
I think `output-path` is simpler for users who don't know fastlane.
Also, some users may write `use: yukiarrr/ios-build-action@master` when using `ios-build-action`.
So I think we should leave `output-path` as it is and split it internally into a directory name and a file name.

```suggestion
        output_directory: File.dirname(ENV["OUTPUT_PATH"]),
        output_name: File.basename(ENV["OUTPUT_PATH"]),
```

Reply:
@yukiarrr This sounds good. Considering backward compatibility, this can be done too.